src/device-monitoring/config/constants.py

```diff
@@ -23,6 +23,7 @@ class Config:
     CACHE_DEFAULT_TIMEOUT = THIRTY_MINUTES
     CACHE_KEY_PREFIX = 'device-monitoring'
     CACHE_REDIS_URL = os.getenv('REDIS_URL_PROD')
+    REDIS_URL = os.getenv('REDIS_URL')
```

Comment:
Do we have this environmental variable registered in our .env files? Is it a new one or a pre-existing one?

Reply:
I've confirmed that we have `REDIS_URL` and also found that we do not have `REDIS_URL_PROD` in our configs. Could we stick to using `REDIS_URL` given that we may need to run the app in other environments other than prod?